### ISSUE TITLE
Failed to build

### ISSUE THREAD

**Issue by ice1000** [eskatos/gradle-command-action#20]
Hi, I'm using your tool and I failed to build my gradle project:

```
 Run eskatos/gradle-command-action@v11s
    JAVA_HOME_11.0.7_x64: C:\hostedtoolcache\windows\jdk\11.0.7\x64
Run eskatos/gradle-command-action@v1
  with:
    arguments: check --info --no-daemon --stacktrace
  env:
    JAVA_HOME: C:\hostedtoolcache\windows\jdk\11.0.7\x64
    JAVA_HOME_11.0.7_x64: C:\hostedtoolcache\windows\jdk\11.0.7\x64
(node:4488) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
##[error]The "path" argument must be of type string. Received type undefined
```

See https://github.com/JetBrains/Arend/runs/768528069?check_suite_focus=true
Revision: https://github.com/JetBrains/Arend/tree/cadb216982eef572f8d9ba6a6c754806a0986678

How could I fix it?

**Comment by badsyntax** [comment]
Like like same issue as: https://github.com/eskatos/gradle-command-action/issues/7#issuecomment-643730292

**Comment by badsyntax** [comment]
You can use a previous version whilst a fix is published, see: https://github.com/eskatos/gradle-command-action/issues/7#issuecomment-643746534

**Comment by eskatos** [comment]
Thanks for the report @ice1000.

Fixed in `v1.2.1` in https://github.com/eskatos/gradle-command-action/commit/c839ac993c41befbe443ddba91a8aafbb5d1f2fb to which `v1` points now.

**Comment by ice1000** [comment]
Fix confirmed
